src/components/Container/Container.scss

```
@@ -0,0 +1,25 @@
+@import './../../styles/fonts.scss';
+@import './../../styles/breakpoints.scss';
+@import './../../styles/alignment.scss';
+
+.container {
+  display: grid;
+  max-width: $breakpoint-laptop-large;
+  padding: 5px;
+  margin: 0 16px;
+  text-align: center;
+}
+// TODO include an escape before this loop
+$total-columns: 12;
+@for $i from 1 through $total-columns {
+  .col__#{$i} {
+    grid-auto-flow: row;
+    grid-template-columns: repeat(#{$i}, 1fr);
+    @media screen and (max-width: $breakpoint-tablet) {
```

Adds space so its easier to read
```suggestion
    grid-template-columns: repeat(#{$i}, 1fr);
    
    @media screen and (max-width: $breakpoint-tablet) {
```

---

@mattlaguardia space added before media queries.